opentracing/src/test/scala/io/skadi/opentracing/SkadiOpentracingTracerSpec.scala

```diff
@@ -95,30 +100,32 @@ class SkadiOpentracingTracerSpec extends SkadiSpec {
 
     val Some(spanContext: OpentracingContext) = tracer
       .trace("op_name") {
-        tracer.getCarrier.map(_.get).flatMap(carrier => tracer.fromCarrier(carrier))
+        tracer.addBaggageItem("test", "foobar") >>
+          tracer.getCarrier.map(_.get).flatMap(carrier => tracer.fromCarrier(carrier))
       }
       .run(None)
       .unsafeRunSync()
+      ._2
 
     val finishedSpan = mockTracer.finishedSpans().get(0)
     spanContext.context.toSpanId shouldBe finishedSpan.context().toSpanId
     spanContext.context.toTraceId shouldBe finishedSpan.context().toTraceId
-
+    jEntiryIterableToScalaMap(finishedSpan.context().baggageItems()) should contain("test" -> "foobar")
```

Comment:
Not 100% sure if this test is now correct.

If one does just plain `spanContext.context.baggageItems shouldBe finishedSpan.context.baggageItems` it will fail with the `spanContext` having empty baggage items.